modules/virt-example-node-placement-node-selector-hyperconverged-cr.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virt/install/virt-specifying-nodes-for-virtualization-components.adoc
+
+[id="virt-example-node-placement-node-selector-hyperconverged-cr_{context}"]
+= Example: Node placement with nodeSelector in the HyperConverged Cluster CR
+
+In this example, `nodeSelector` is configured so that infrastructure resources are placed on nodes that are labeled with `nodeType = infra` and workloads are placed on nodes labeled with `nodeType = nested-virtualization`.
```

Comment:
please use another nodeType - we do not want to create the impression that we support nested virt.

Reply:
After discussing this with @nunnatsa, I am going to change most of the key/value pairs to generic examples.